### PR TITLE
Get Player from CLI

### PR DESCRIPTION
**Pull request by FarzanRashid**
This PR contains necessary changes to simulate a roulette strategy provided by user through CLI

### CODE REVIEW

File: src/player_factory.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from players.martingale import Martingale
+from players.cancellation import PlayerCancellation
+from players.fibonacci import PlayerFibonacci
+from players.seven_reds import SevenReds
+from players.random import PlayerRandom
+from players.passenger57 import Passenger57
+from players.player1326.player1326 import Player1326
+
+
+def provide_player(player_name: str, table: Table, wheel: Wheel) -> Player:
```
Comment (FarsanRashid) [inline src/player_factory.py:13]: rename method to `player_factory` or `get_player`

File: tests/test_strategy_factory.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    def test_provide_player_provides_player(self):
+        martingale_player = provide_player("Martingale", self.table, self.wheel)
+        self.assertIsInstance(martingale_player, Martingale)
+
+        sevenreds_player = provide_player("Sevenreds", self.table, self.wheel)
+        self.assertIsInstance(sevenreds_player, SevenReds)
+
+        fibonacci_player = provide_player("Fibonacci", self.table, self.wheel)
+        self.assertIsInstance(fibonacci_player, PlayerFibonacci)
```
Comment (FarsanRashid) [inline tests/test_strategy_factory.py:26]: some player creation needs only table parameter. test for at least one of them.

File: tox.ini [hunk 3]
```diff
@@ -20,7 +20,7 @@ commands =
 [testenv:lint]
 commands =
    pylint src --disable=missing-module-docstring,invalid-name,too-few-public-methods,\
-   redefined-builtin,too-many-lines src
+   redefined-builtin,too-many-lines,no-value-for-parameter src
```
Comment (FarsanRashid) [inline tox.ini:23]: `no-value-for-parameter` is an important check ignoring which for whole application can break things. better ignore on local scope where main is called.

`main() # pylint: disable=no-value-for-parameter`

Comment (FarsanRashid) [reply tox.ini:23]: lint gets perfect score without `too-many-lines`. better remove it. probably some refactoring fixed too many line issues at some point.

**Comment by FarsanRashid** [comment]
Pull request comment should be concise. "This PR contains necessary changes to simulate a roulette strategy provided by user through CLI" is verbose and could have been like  "This PR adds support for getting player name from CLI"

File: tox.ini (hunk 3 above)
Comment (FarzanRashid) [reply tox.ini]: Done.

File: tests/test_strategy_factory.py (hunk 2 above)
Comment (FarzanRashid) [reply tests/test_strategy_factory.py]: All these existing players only need table parameter. Added tests for those who needs table and wheel both.

File: tests/test_strategy_factory.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+from table import Table
+from wheel import Wheel
+from bin_builder import BinBuilder
+from player_factory import provide_player
+from players.martingale import Martingale
+from players.fibonacci import PlayerFibonacci
+from players.seven_reds import SevenReds
+
+
+class TestPlayerProvide(TestCase):
```
[the remark this answers is not in the document]
Comment (FarzanRashid) [reply tests/test_strategy_factory.py]: Done.

File: src/player_factory.py (hunk 1 above)
Comment (FarzanRashid) [reply src/player_factory.py]: Done.

**Pull request merged**